greek/offer_marshmallow_a_cigar/intimacy/intimacy.md

```diff
@@ -0,0 +1,2 @@
+Το σκεφτεσαι περισσοτερο αναβωντας ακομα ένα τσιγαρο.
+Το ζαχαρωτο χερι εχει πνιγει στους καπνους αλλα πλεον δεν σε νοιαζει, εισαι οικειος μαζι του αφου εχουν περασει πεντε ολοκληρα λεπτα από τοτε που γνωριστηκατε. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.